docs/quick-start.md

```diff
@@ -415,26 +415,92 @@ You can find an additional unary client and server example (a runnable one) in t
 
 ##### Server Streaming Example
 
-Simply return a [core.async](https://github.com/clojure/core.async)
-[channel](https://clojuredocs.org/clojure.core.async/chan) in the :body key returned by your interceptor handler
-instead of a map as above in the unary example
+When a client sends a request to the server, two [channels](https://clojuredocs.org/clojure.core.async/chan) are provided in the request: `:grpc-out` and `close-ch`.
+
+* `:grpc-out` channel 
+
+Is the streaming channel, used to send all the messages. When the server is done with the streaming, simply 
```

Comment:
The only thing I would suggest is we reword the "close! the channel and return it in the body" as it sounds like that is explicit ordering and the fact is, the body should be returned before the streaming begins.

Maybe something like 

"Is the streaming channel, used to send all the messages. The handler first acknowledges streaming will start by returning the same grpc-out channel as the :body of the response map.  When the server is done..."